### ISSUE TITLE
[BUG] Shared job vehicle keys does not work for custom added cars

### ISSUE THREAD

**Issue by ChickenDipper1** [qbcore-framework/qb-vehiclekeys#164]
**Describe the bug**
Other players that are for example police do not get shared keys to custom cars that are added to the server and added to the relevant job shared keys in vehicle-keys config.lua.
The cars are added properly and are available to buy in qb-vehicleshop.

**To Reproduce**
1. Add custom vehicle to resources and qbcore/shared/vehicles.lua
2. Add to vehiclekeys Config.SharedKeys
3. Spawn car
4. Other player in police job has to hotwire the car

**Expected behavior**
Custom cars should have key sharing applied when added in vehicle keys config.

**Screenshots**
N/A

**Questions (please complete the following information):**
 - When you last updated: [e.g. last week] Today
 - Are you using custom resource? which ones? [e.g. zdiscord, qb-target] No
 - Have you renamed `qb-` to something custom? [e.g. yes/no] No

**Additional context**
N/A

**Comment by tom-osborne** [comment]
@ChickenDipper1 
I suspect this may be because your addon vehicle might have lower case letters in the `vehicle.meta` file entry for `<modelName>` perhaps?
In this case, this line will be causing the issue: https://github.com/qbcore-framework/qb-vehiclekeys/blob/main/client/main.lua#L304

Since all default GTA vehicles will return uppercase for the modelName when calling the following code:
```lua
local vehName = GetDisplayNameFromVehicleModel(GetEntityModel(veh))
```
I wonder if it is returning lowercase letters for addon vehicles.

You can test this by printing `vehName` after this line and seeing if there are any lowercase letters: https://github.com/qbcore-framework/qb-vehiclekeys/blob/main/client/main.lua#L296

**Comment by ChickenDipper1** [comment]
Yep that was exactly the cause and changing the <modelName> of each custom car to uppercase fixes the issue.

Thank you!

**Comment by tom-osborne** [comment]
@ChickenDipper1 great, thanks for testing and the feedback. I will implement a PR to protect against this in future. Allowing any addons to have lowercase. In the mean-time, as you suggested, changing your addon vehicles to uppercase will fix it.

**Comment by tom-osborne** [comment]
Fixed in #166

**Comment by andrezote92** [comment]
Hey! the solution only can work in two vehicles. My list have 7 police cars, but doing this solution, only can work 2
